target/linux/octeon/Makefile

```diff
@@ -21,6 +21,6 @@ endef
 
 include $(INCLUDE_DIR)/target.mk
 
-DEFAULT_PACKAGES += mkf2fs e2fsprogs
+DEFAULT_PACKAGES += mkf2fs e2fsprogs f2fs-tools
```

Comment:
This is unrelated and should go in a different commit, if it should go in at all.

Reply:
Tbh im not sure if it should go in but its the only way i got this working because the overlay filesystem kept failing.

Reply:
That's strange. I'm using the squashfs + f2fs image on my ERLite and it works fine. I assume you are testing on master, as the PR is against master? Can you give more info on how it fails?

Reply:
When i leave that package out it says "overlay filesystem has not been fully initialized yet." and then it switches to a temporary one that does hold the data until next reboot. I traced it down to the missing mkfs binary for f2fs. As soon as it was available it worked.

Reply:
But f2fs-tools doesn't contain the mkfs utility:
```
root@ar1:~# opkg files f2fs-tools
Package f2fs-tools (1.8.0-3) is installed on root and has the following files:
/usr/sbin/f2fstat
/usr/sbin/parse.f2fs
/usr/sbin/fibmap.f2fs
```

Reply:
Maybe you just didn't have mkf2fs enabled in your config initially? Having something in DEFAULT_PACKAGES does not guarantee it will be enabled in the .config. I suggest to disable f2fs-tools in menuconfig again and try a new build. After seeing what files are in there, it definitely does not belong in DEFAULT_PACKAGES.

I'll gladly test on my ERL when you update the PR.

Reply:
I will do everything you mentioned above and update this pr.
I will also check if the mkfs.f2fs gets compiled and is available to the system.
I verified it got enabled in my configs.